includes/database/tables/class-order-transactions.php

```diff
@@ -139,4 +141,21 @@ protected function __202105291() {
 		return true;
 	}
 
+	/**
+	 * Upgrade to version 202205241
+	 * 	- Add combined index for object_type, object_id.
+	 *
+	 * @since 3.0
+	 * @return bool
+	 */
+	protected function __202205241() {
+		if ( $this->index_exists( 'object_type_object_id' ) ) {
+			return true;
+		}
+
+		$this->get_db()->query( "ALTER TABLE {$this->table_name} ADD INDEX object_type_object_id (object_type, object_id)" );
```

Comment:
Looking at the other upgrade functions in the class, shouldn't this be this instead?

```php
		return $this->is_success(
			$this->get_db()->query( "ALTER TABLE {$this->table_name} ADD INDEX object_type_object_id (object_type, object_id)" )
		);
```
and remove the final `return true;`?

Reply:
Every time we're altering for an index, we use the `return true` method of doing this. I pulled this from other upgrade routines that specifically worked around altering to add an index, not a column specifically, so there might be a difference here.